installer/upgrader/consoleTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
```

Comment:
hope we are going to remove this once we complete the upgrader

Reply:
removed